css/src/components/popover.scss

```diff
@@ -21,7 +25,6 @@ $popover-width: 224px !default;
 		border-radius: $popover-border-radius;
 		background-color: $popover-background-color;
 		box-shadow: $popover-shadow;
-		overflow: hidden;
```

Comment:
_Reviewers note:_ removed the `overflow: hidden;` to cover those cases when one popover exists in another one to make its content visible.